neuro_evolution_ctrnn/brain_visualizer/weights.py

```diff
@@ -122,6 +125,10 @@ def draw_maximum_weights(visualizer: "brain_visualizer.BrainVisualizer", start_p
     @staticmethod
     def draw_weights(visualizer: "brain_visualizer.BrainVisualizer", start_pos_dict: dict, end_pos_dict: dict,
                      weight_matrix, is_input: bool = False, in_values: np.ndarray = None) -> None:
+        # TODO ich glabue dass die Matrix diagnal gespiegelt war, war ein großer Bug, der nicht aufgefallen ist,
+        ## weil weniger input als Neuronen da sind,
+        ## W quadratisch ist und die Methode für den Output verkehrt herum aufgerufen wurde
```

Comment:
Code Comments fände ich in Englisch besser

Reply:
erledigt